topology.yaml

```diff
@@ -0,0 +1,27 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# yaml definition file for the nodebase Kerberos topology.
+
+name: http_kerberos
+description: A HTTP Server topology with Kerberos Authentication
+
+node groups:
+    kdc-node:
+        - kdc
+    webserver-node:
+        - webserver
+
+start args:
+    --kerberos-config-directory:
```

Comment:
I based this out of the existing topology_nodebase_kerberos and thought it would be useful to keep this argument.  You are right, that this is not really useful and can remove it.